### ISSUE TITLE
AVT 1 - Modal has DAP violation for missing accessible name

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2475]
Environment
macOS Mojave version 10.14.4
Version 74.0.3729.108 (Official Build) (64-bit)

Detailed Description
Run DAP with Feb. 2019 ruleset on the Modal Component
The following violation is reported:
"An interactive element/widget must have an accessible name"
WAI-ARIA Specification for Widget Roles
WAI-ARIA Specification for Accessible Name Computation

To fix the violation, replace the button title="Close the modal" with aria-label="Close the modal" as shown in the screenshot below:
![Screen Shot 2019-04-30 at 10 46 19 AM](https://user-images.githubusercontent.com/21676914/56975008-8f1d1f80-6b35-11e9-87b4-b02466c85008.png)

See original issue #2192

**Comment by snidersd** [comment]
@dakahn Retested this today and there are no violations. Confirmed the aria-label has been added, see screenshot below.
![Screen Shot 2019-05-14 at 10 09 09 AM](https://user-images.githubusercontent.com/21676914/57709704-25bb0780-7631-11e9-9602-346e75689bcc.png)
